libs/langchain/langchain/agents/xml/base.py

```diff
@@ -97,3 +102,70 @@ async def aplan(
         }
         response = await self.llm_chain.acall(inputs, callbacks=callbacks)
         return response[self.llm_chain.output_key]
+
+
+def create_xml_agent(
+    llm: BaseLanguageModel, tools: Sequence[BaseTool], prompt: ChatPromptTemplate
```

Comment:
No reason this one can't be `BasePromptTemplate`?

https://smith.langchain.com/hub/hwchase17/xml-agent-convo

Reply:
your right